apps/calendar/test/unit/support/wait_for.js

```diff
@@ -9,7 +9,7 @@ function waitFor(test, callback) {
     return callback();
   }
 
-  setTimeout(() => waitFor(test, callback), 500);
+  setTimeout(() => waitFor(test, callback), 200);
```

Comment:
this wasn't supposed to be on the PR but tests was taking too long with 500ms :D